src/components/inspect-panel-info.animal.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import { BackpackMouseType, InspectContext } from "../models/backpack-mouse";
+import { speciesDef } from "../models/units";
+import { renderGameteIcons } from "./inspect-panel";
+
+export function renderAnimalInfo(
+    org: BackpackMouseType, context: InspectContext, isGamete: boolean, showGenotype: boolean): JSX.Element {
+  const species = speciesDef(org.species);
+  const sexLabel = org.sex === "female" ? "Female" : "Male";
+  const colorLabel = species.getPhenotypeLabel(org.phenotype);
+  const genotypeLabel = species.getGenotypeHTMLLabel(org.genotype);
+  const rowClass = "info-row" + (context === "nest" ? "" : " wide");
+  const mouseInfoClass = "mouse-info" + (context === "population" ? " no-header" : "");
+  const infoTypeClass = "info-type" + (context === "population"  ? " population " : "");
```

Comment:
Not sure if string template literals would make these easier to write/read.  I've tended to favor them more recently when creating CSS classes.

Reply:
Or a library like `classNames` or `clsx`, which improve readability.